src/helpers/script_helpers.sh

```diff
@@ -14,6 +14,7 @@ function command_steps {
   echo_if_all_true "change_directory $(git_root)" "$RUN_IN_GIT_ROOT" "$IN_SUB_FOLDER"
   echo_if_all_true "stash_open_changes" "$STASH_OPEN_CHANGES" "$INITIAL_OPEN_CHANGES"
   steps
+  echo_if_all_true "set_previous_branch \$(expected_previous_branch)"
```

Comment:
If the expected previous branch isn't computed until after the rest of the steps are run then it definitely won't work for abort/continue. What about if this was
```
set_previous_branch $INITIAL_BRANCH_NAME $PREVIOUS_BRANCH_NAME
```
That way we don't defer execution of a function but just move that logic to set_previous_branch (or a different function if you want to keep the current flow.